README.md

```diff
@@ -568,6 +568,25 @@ func main() {
 		}
 	})
 
+	// Example for binding XML (
+	//	<?xml version="1.0" encoding="UTF-8"?>
+	//	<root>
+	//		<user>user</user>
+	//		<password>123</user>
+	//	</root>)
+	router.POST("/loginXML", func(c *gin.Context) {
+		var xml Login
+		if err := c.ShouldBindXML(&xml); err == nil {
+			if xml.User == "manu" && xml.Password == "123" {
+				c.JSON(http.StatusOK, gin.H{"status": "you are logged in"})
+			} else {
+				c.JSON(http.StatusUnauthorized, gin.H{"status": "unauthorized"})
+			}
+		} else {
+			c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		}
```

Comment:
```go
    if err := c.ShouldBindXML(&xml); err != nil {
      c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
      return
    }
    
    if xml.User != "manu" || xml.Password != "123" {
      c.JSON(http.StatusUnauthorized, gin.H{"status": "unauthorized"})
      return
    } 
    
    c.JSON(http.StatusOK, gin.H{"status": "you are logged in"})
```

more readable.

Reply:
agreed with @appleboy

Reply:
make a new pr change these case? @appleboy

Reply:
@syssam Why need to make a new PR? I think you can change in this PR. otherwise LGTM

Reply:
@syssam you should change the code and git commit and git rebase your commit message, not new pull request.